tests/manage/storageclass/test_new_sc_rbd_e2e_workloads.py

```diff
@@ -0,0 +1,88 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import E2ETest, tier2
+
+
+@tier2
```

Comment:
Please implement code to provide a comparison between with and without compression for each of the workloads